python/rtypes/dataframe/trigger_manager.py

```diff
@@ -0,0 +1,72 @@
+
+from enums import ObjectType
+import os
+
+
+class TriggerManager(object):
+    """ TriggerManager regulates the triggers in the dataframe:
+        - giving a space for triggers to live (See other managers for an approach to this)
+        - allowing users to add triggers into the manager
+        - allowing users to delete triggers into the manager
+        - preventing triggers of the same time and action from existing
+        - allow the user to add triggers with a specified priority and have that priority take affect with any overlapping triggers
```

Comment:
Keep the length of any line not more than 80 columns wide. It's not being enforced in this whole project yet. But I intend to do that in this reorg. So it would be good if all the code and comments you wrote dont exceed 80 columns.